src/offline/cable_mpimaster.F90

```diff
@@ -391,48 +342,8 @@ SUBROUTINE mpidrv_master (comm, trunk_sumbal)
           ENDIF
 
           IF ( TRIM(cable_user%MetType) .EQ. 'plum' ) THEN
-             ! CLN HERE PLUME modfications
-             IF ( CALL1 ) THEN
-                CALL PLUME_MIP_INIT( PLUME )
-                dels      = PLUME%dt
-                koffset   = 0
-                leaps = PLUME%LeapYears
-                WRITE(str1,'(i4)') CurYear
-                str1 = ADJUSTL(str1)
-                WRITE(str2,'(i2)') 1
-                str2 = ADJUSTL(str2)
-                WRITE(str3,'(i2)') 1
-                str3 = ADJUSTL(str3)
-                timeunits="seconds since "//TRIM(str1)//"-"//TRIM(str2)//"-"//TRIM(str3)//" &
-                     00:00"
-
-             ENDIF
              kend = NINT(24.0*3600.0/dels) * LOY
           ELSE IF ( TRIM(cable_user%MetType) .EQ. 'cru' ) THEN
-             ! CLN HERE CRU modfications
-             IF ( CALL1 ) THEN
-
-                CALL CPU_TIME(etime)
```

Comment:
I couldn't see the corresponding calls to `CPU_TIME` in the new code. Did they disappear or is it github confusing me?

Reply:
The calls to `CPU_TIME(etime)` were removed from the initialisation stages as the value of `etime` was not used in any of the drivers - `etime` was only used after a subsequent call to `CPU_TIME` which overwrites the previous value of `etime` from the initialisation stage so they were safe to remove.